lib/ansible/modules/identity/ipa/ipa_dnsrecord.py

```diff
@@ -99,6 +99,16 @@
     record_type: 'AAAA'
     record_ip: '::1'
 
+# Ensure a PTR record is present
+- ipa_dnsrecord:
+    ipa_host: spider.example.com
+    ipa_pass: Passw0rd!
+    state: present
+    zone_name: 2.168.192.in-addr.arpa
+    record_name: 5
+    record_type: 'PTR'
+    record_ip: 'internal.ipa.example.com'
```

Comment:
+1 for `record_value`. Can you add record_value as AnsibleModule Argument? and make dependency to `record_type` 

```
--- a/lib/ansible/modules/identity/ipa/ipa_dnsrecord.py
+++ b/lib/ansible/modules/identity/ipa/ipa_dnsrecord.py
@@ -207,7 +207,8 @@ def main():
             zone_name=dict(type='str', required=True),
             record_name=dict(type='str', required=True, aliases=['name']),
             record_type=dict(type='str', required=False, default='A', choices=record_types),
-            record_ip=dict(type='str', required=True),
+            record_ip=dict(type='str', required=False),
+            record_value=dict(type='str', required=False),
             state=dict(type='str', required=False, default='present', choices=['present', 'absent']),
             ipa_prot=dict(type='str', required=False, default='https', choices=['http', 'https']),
             ipa_host=dict(type='str', required=False, default='ipa.example.com'),
@@ -217,6 +218,10 @@ def main():
             validate_certs=dict(type='bool', required=False, default=True),
         ),
         supports_check_mode=True,
+        required_if=[('record_type', 'PTR', ['record_value']),
+                     ('record_type', 'A', ['record_ip']),
+                     ('record_type', 'AAAA', ['record_ip']),
+                     ],
     )
```

And Also add documentation and example for this change.